templates/main/index.html.ep

```diff
@@ -19,7 +19,6 @@
     </div>
 </div>
 
-<a name="build-results"/>
```

Comment:
why would you need to have a wrapping <div> if all you need is the place in front? Just use <a id='aa'> - the name attribute is the only thing invalid to HTML5